### PR TITLE
Create t4fs.md

### PR DESCRIPTION
**Pull request by allysonlister**
Added as part of the work for #1520

### CODE REVIEW

**Comment by jamesaoverton** [comment]
The automated checks did not run for my last commit. <https://www.githubstatus.com/> says that "Actions is experiencing degraded performance".

**Comment by cthoyt** [comment]
needs a `preferredPrefix` and to use an appropriate `domain`

File: ontology/t4fs.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+layout: ontology_detail
+id: t4fs
+title: terms4FAIRskills
+contact:
+  email: pete_mcq@protonmail.com
+  github: drosophilic
+  label: Peter McQuilton
+  orcid: 0000-0003-2687-1982
+description: A terminology for the skills necessary to make data FAIR and to keep it FAIR.
+domain: information
```
Comment (matentzn) [inline ontology/t4fs.md:11]: changed that.

**Comment by matentzn** [comment]
@allysonlister can you review my changes - they were needed to pass the file!

**Comment by cthoyt** [comment]
before putting @drosophilic as the contact, would be good to get a confirmation from him that he knows that he is responsible for github inquiries on this

File: ontology/t4fs.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+homepage: https://github.com/terms4fairskills/FAIRterminology
+license:
+  label: CC BY 4.0
+  url: https://creativecommons.org/licenses/by/4.0/
+preferredPrefix: TF4S
+products:
+- id: t4fs.owl
+- id: t4fs.obo
+- id: t4fs.json
+- id: t4fs-community.owl
```
Comment (cthoyt) [inline ontology/t4fs.md:21]: can you add "title" keys for all of these so we can tell what the difference is between the base ones and the `-community` ones?

**Review by matentzn — CHANGES_REQUESTED** [review]
- [ ] confirm the two edits performed by me
- [ ] Address @cthoyt suggestions on for better descriptions of release artefacts

**Comment by allysonlister** [comment]
I have done the first two you requested @matentzn - hopefully it looks OK as the titles look a bit repetitive but I couldn't figure out a way to have one title for all community formats.

I have emailed @drosophilic and am waiting to hear back from him, as it may be best to switch our names, I'll update once I know more. Thanks!

**Comment by Drosophilic** [comment]
Hello! Apologies for the radio silence, I was snowed under at work. Allyson has done a great deal of work on this and I think it makes perfect sense for @allysonlister to be the OBO contact.

**Comment by allysonlister** [comment]
Super, thank you! I've updated the contact details to me. Is there anything left to do now, as I believe I've done everything requested?

Thanks very much!

**Review by cthoyt — APPROVED** [review]

**Review by matentzn — APPROVED** [review]

**Pull request merged**